<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;MATRIZ A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantidadde de Linhas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLinhasMatA"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>222</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Quantidade de Colunas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditColunasMatA"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarMatB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;MATRIZ B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>222</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Quantidade de Colunas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditColunasMatB"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Quantidadde de Linhas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLinhasMatB"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaMatA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaidaMatB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdicionar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubtrair">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicacaoPorK">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTransposta">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPotencia">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Triangular Superior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triangular Inferior</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Simetrica</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Identidade</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Iguais</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Diferentes</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ortogonal</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Permutação</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>MATRIZ A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MATRIZ B</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
